app/install.php

```diff
@@ -122,6 +122,7 @@ function saveStep1() {
 function saveStep2() {
 	$user_default_config = Minz_Configuration::get('default_user');
 	if (!empty($_POST)) {
+		$_SESSION['title'] = 'FreshRSS';
```

Comment:
It would be nicer to get the value from https://github.com/marienfressinaud/FreshRSS/blob/fix/remove-requirement-on-title/data/config.default.php#L25 with something like `$system_conf->title`